frontend/src/server.js

```diff
@@ -4,7 +4,6 @@ import express from 'express'
 import requestLanguage from 'express-request-language'
 import { renderToString } from 'react-dom/server'
 import { ServerLocation } from '@reach/router'
-import { Logger } from '@cdssnc/logdriver'
```

Comment:
Wait, aren’t we still using Logger down near line 93?

Reply:
yeah, I get an error in the console
<img width="435" alt="image" src="https://user-images.githubusercontent.com/8228248/52497988-8a377800-2ba5-11e9-8c79-fe7c77c5c716.png">

Reply:
Ah, same error in master. Basically, the issue is that in that code `Logger` is being used in a script being sent to the client, not run on the server where the import is happening. So I think we want to not use Logger in that script (and delete the import as you already have)